printrun/pronterface.py

```diff
@@ -81,35 +81,38 @@ def format_length(mm, fractional=2):
     return '%%.%df' % fractional % units + suffix
 
 class ConsoleOutputHandler:
-    """Handle console output. All messages go through the logging submodule. We setup a logging handler to get logged messages and write them to both stdout (unless a log file path is specified, in which case we add another logging handler to write to this file) and the log panel.
-    We also redirect stdout and stderr to ourself to catch print messages and al."""
+    """Handle console output. All messages go through the logging submodule. We setup a logging handler to get logged
+    messages and write them to both stdout (unless a log file path is specified, in which case we add another logging
+    handler to write to this file) and the log panel. We also redirect stdout and stderr to ourselves to catch print
+    messages and al."""
 
-    def __init__(self, target, log_path):
+    def __init__(self, target, log_path, log_stdout):
         self.stdout = sys.stdout
         self.stderr = sys.stderr
         sys.stdout = self
         sys.stderr = self
-        self.print_on_stdout = not log_path
-        if log_path:
```

Comment:
I was worried about removing this check, but in my tests I could not make it fail due to this removal. So I'm happy with it. But if you could double check I would appreciate it.

Reply:
it was redundant: there is already the same [check][1] in `setup_logging()`

[1]: https://github.com/kliment/Printrun/blob/424a3ec5ab0e8ec40624ded34775c54dd0a628c9/printrun/utils.py#L84